### ISSUE TITLE
SkeletonPlaceholder.js seems to be importing "@react-native-community/masked-view" @ v4.0.0.

### ISSUE THREAD

**Issue by sophie-li** [chramos/react-native-skeleton-placeholder#41]
Got the following error on version `4.0.0`

`Unable to resolve module @react-native-community/masked-view from ... /lib/SkeletonPlaceholder.js: @react-native-community/masked-view could not be found within project`

![Screen Shot 2021-05-03 at 5 50 52 PM](https://user-images.githubusercontent.com/8574342/116938016-1d05d680-ac38-11eb-820a-00af8abd6cd2.png)

Version: 4.0.0

**Comment by Sayon1992** [comment]
Yeah, i've the same problem. expecting them to change to masked-view sice it still has performance issues.

**Comment by chramos** [comment]
Hey guys, I'll check this

**Comment by diazdell91** [comment]
+1

**Comment by okimotoren** [comment]
hello, I had the same situation", 
and i changed here

before : `const masked_view_1 = __importDefault(require("@react-native-community/masked-view"));`
after : `const masked_view_1 = __importDefault(require("@react-native-masked-view/masked-view"));`

#37
<img width="761" alt="Screen Shot 2021-05-18 at 16 37 08" src="https://user-images.githubusercontent.com/65141910/118610967-4b1e1580-b7f7-11eb-93d2-6a25ce2297cc.png">

**Comment by okimotoren** [comment]
Hello, I had the same situation, and i solved it by the change below.

### Solution:
in node_modules/react-native-skeleton-placeholder/lib/SkeletonPlaceholder.js

<img width="751" alt="Screen Shot 2021-05-18 at 16 53 26" src="https://user-images.githubusercontent.com/65141910/118613280-920d0a80-b7f9-11eb-98b9-7de1b66ccad1.png">

**before** : `const masked_view_1 = __importDefault(require("@react-native-community/masked-view"));`
**after** : `const masked_view_1 = __importDefault(require("@react-native-masked-view/masked-view"));`



I guess it was happened with #37 .

I hope it could help someone.
@sophie-li  @chramos

**Comment by rdgoutiyama** [comment]
I solved this issue with:

First, try to install:

```
yarn add @react-native-masked-view/masked-view
```

And run project.  If an error occurs, like: 

_While trying to resolve module react-native-skeleton-placeholder from file xxxxx, ...._

I make this pull request #50

**Comment by andreialecu** [comment]
I think the `npm publish` was somehow messed up because:

https://cdn.jsdelivr.net/npm/react-native-skeleton-placeholder@4.0.0/lib/SkeletonPlaceholder.js (search for `react-native-community` here)

Does not match the source of https://github.com/chramos/react-native-skeleton-placeholder/blob/master/src/SkeletonPlaceholder.tsx.

@chramos mind publishing the library again?

**Comment by shikharsrivastava23** [comment]
Can someone clarify if we need @react-native-masked-view/masked-view or @react-native-community/masked-view? Having both in the package.json is causing mergeDex error when building for android.

**Comment by shikharsrivastava23** [comment]
> Can someone clarify if we need @react-native-masked-view/masked-view or @react-native-community/masked-view? Having both in the package.json is causing mergeDex error when building for android.

I used @react-native-community/masked-view, seems to work fine for now.

**Comment by kimsean** [comment]
> Can someone clarify if we need @react-native-masked-view/masked-view or @react-native-community/masked-view? Having both in the package.json is causing mergeDex error when building for android.

I have the same issue.

**Comment by MatiSera** [comment]
Hi! This fix will be uploaded? Because I'm still facing this error.

**Comment by andreialecu** [comment]
Version 5.0 uses `@react-native-masked-view/masked-view`. You will need to remove the older (community) one.

**Comment by stoyan-frieze** [comment]
I think I am having conflict issues between @react-native-community/masked-view and @react-native-masked-view/masked-view - '**Invariant Violation: Tried to register two views with the same name RNCMaskedView, js engine: hermes**' But I am using react-navigation which has @react-native-community/masked-view as a dependency so I can't remove it. Can they coexist? Surely, I am not the only person using react navigation.

**Comment by MaganAnkur** [comment]
I am getting the same error in version 5.

I have followed the below step

Installed Masked view and linear gradient libraries before

Below is SkeletonPlaceholder.js

const masked_view_1 = __importDefault(require("@react-native-masked-view/masked-view"));
const react_native_linear_gradient_1 = __importDefault(require("react-native-linear-gradient"));

But still I am getting below : 
Error: Unable to resolve module `@react-native-community/masked-view` from `node_modules/react-native-skeleton-placeholder/lib/SkeletonPlaceholder.js`: @react-native-community/masked-view could not be found within the project.

**Comment by felansu** [comment]
more 1 here 🪂

**Comment by Choyeongdeok** [comment]
> I think I am having conflict issues between @react-native-community/masked-view and @react-native-masked-view/masked-view - '**Invariant Violation: Tried to register two views with the same name RNCMaskedView, js engine: hermes**' But I am using react-navigation which has @react-native-community/masked-view as a dependency so I can't remove it. Can they coexist? Surely, I am not the only person using react navigation.

I removed @react-native-masked-view/masked-view and restart then fix error

**Comment by sanoobpookodan** [comment]
Note: This package requires @react-native-masked-view/masked-view and react-native-linear-gradient
